packages/eslint-plugin/docs/rules/i18n-no-collapsible-whitespace.md

```diff
@@ -0,0 +1,17 @@
+# Disallow collapsible whitespace in translatable strings. (i18n-no-collapsible-whitespace)
+
+Translatable strings that consist of nothing but a placeholder are rather pointless and not really translatable.
```

Comment:
This description doesn't seem to encompass the full scope of the rule, where in the implementation I see various consideration for newlines, tabs, consecutive spaces. I assume this is also related to why the rule needed to be disabled for Gutenberg. Can you expand on this documentation? 

To the general point about disabling this for Gutenberg: Can you clarify if those instances within Gutenberg code are errors which need to be improved, or valid use-cases? If it's the latter, I wonder if that contradicts the purpose of the rule altogether.

Reply:
Oh, looks like I didn't update the text after copy/pasting 🙂

Reply:
> Can you clarify if those instances within Gutenberg code are errors which need to be improved, or valid use-cases? If it's the latter, I wonder if that contradicts the purpose of the rule altogether.

So there are only 5 instances:

https://github.com/WordPress/gutenberg/blob/3a675f5fce3b9ff43630a1b738f51a2bae805464/packages/block-editor/src/components/media-upload-progress/index.native.js#L124-L126

https://github.com/WordPress/gutenberg/blob/3a675f5fce3b9ff43630a1b738f51a2bae805464/packages/block-library/src/code/index.js#L25-L32

https://github.com/WordPress/gutenberg/blob/3a675f5fce3b9ff43630a1b738f51a2bae805464/packages/block-library/src/preformatted/index.js#L25-L32

https://github.com/WordPress/gutenberg/blob/3a675f5fce3b9ff43630a1b738f51a2bae805464/packages/block-library/src/verse/index.js#L26-L33

https://github.com/WordPress/gutenberg/blob/3a675f5fce3b9ff43630a1b738f51a2bae805464/packages/editor/src/components/reusable-blocks-buttons/reusable-block-delete-button.js#L69-L76

Not too familiar with those files

Reply:
So are those "bad" ? Would we typically want multiple lines of a translated string to be separate strings?

Reply:
> Would we typically want multiple lines of a translated string to be separate strings?

Related (the opposite): #19048

Reply:
If it's not something we want to deal with, I'd wonder if we should still enable the rule, and just add inline disablings for these occurrences 'til we can sort out what to do with them (to add proper "justification" after the fact).

Reply:
> we should still enable the rule, and just add inline disablings for these occurrences

That was my thinking as well. These particular occurrences here all seem legit to me.

Reply:
Enabled the rule in 948f22aa46ecd4cb750c00460fd7d49f1ed20823